### ISSUE TITLE
Keybind to change betwen default and blind in-game

### ISSUE THREAD

**Issue by Lukkez** [davidmann4/ogar-feeder-bot#429]
the title says all

**Comment by ghost** [comment]
yeah  I have it easy but I use for older version  in new version blind mode doesn't work

**Comment by David1ali12** [comment]
@lukkez add me on skype:    David Ali-09

**Comment by Lukkez** [comment]
@Martinali12 I can't, u can make another thing?

**Comment by ravyn120** [comment]
@problemsofworld I fixed it in newer versions you just need to read the error from update userscript and find in feeder what you need to add as sending pos

**Comment by ravyn120** [comment]
@problemsofworld I would tell you what to add to the userscript but it's pretty simple to figure out

**Comment by ghost** [comment]
@ravyn120  I use older verison wanna trade with 20 working cookies but I want it in new version of orad  ravny120 I have good new codes for trading

**Comment by Lukkez** [comment]
@problemsofworld You can give the script to change between? i use old version with BreakBot coord system

**Comment by ghost** [comment]
yeah add me on skype:lv-0agar1

**Comment by ravyn120** [comment]
@problemsofworld so 20 cookies

**Comment by Lukkez** [comment]
I can't use skype -_-

**Comment by David1ali12** [comment]
@Lukkez  it's 2016 are u using yahoo?

**Comment by Lukkez** [comment]
No lol, i can now

**Comment by Lukkez** [comment]
I wasn't with skype installed
